.github/workflows/frontend.yaml

```diff
@@ -0,0 +1,68 @@
+name: Frontend
+on:
+  push:
+    branches: [main]
+  pull_request:
+    paths:
+      - frontend/**
```

Comment:
Na path frontend/** by to melo byt limitovane i pro deployment (pri push do main branch). Pridal bych i workflow_dispatch, diky kteremu to pujde pustit manualne (muze se to hodit, been there).

Reply:
Jo, to je fakt. Hotovo obojí.